iFixit/react-commerce
# Product Page: Update compatibility linking

Closes [#47433](https://github.com/iFixit/ifixit/issues/47433)

- Uses a `<details>` element instead of a chakra disclosure for compatibility section dropdowns. This gives us the find-in-page feature in chromium browsers.
- Updates compatibility linking to be more intuitive.

## CR

Best viewed with [whitespace hidden and split](https://github.com/iFixit/react-commerce/pull/1604/files?diff=split&w=1).

## QA

- Visit a product with more than 10 model numbers on one of it's devices. For example, `/products/refrigerator-ice-maker-assembly-wr30x10093` in the production preview.
- Make sure both compatibility components meet the spec as described in the [issue](https://github.com/iFixit/ifixit/issues/47433) and on [slack](https://ifixit.slack.com/archives/C01K32B4FPX/p1681837185634129).
- Make sure that on chromium browsers, using find-in-page (Ctrl+f) for a model number in a collapsed compatibility section dropdown opens the dropdown and jumps to the model number.

CC @sterlinghirsh
